RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Attempt to merge union types during schema conversion.
```

Comment:
with the last change this is probably a minor now, since it's a brand new feature (plus a fix) 😊